### ISSUE TITLE
Can't utilize or find TextArea component.

### ISSUE THREAD

**Issue by RookTKO** [department-of-veterans-affairs/vets-design-system-documentation#1470]
# Support question

- [x] I’ve searched for any related issues/questions on github

## How can we help?

I am part of the VANotify team and we are updating our @department-of-veterans-affairs/component-library from 8 to latest (long story on how it got so behind). Anyway, I can't find a TextArea component inside the package but it seems to still show up on story book. Did it totally get removed? I don't see it in the code?

## What you have tried:

I looked through the code, issues and started (and am currently) looking through older versions.

## What problem does this solve?

TechDebt.

## Additional Context

Help me Obi-Wan Kenobi, you are my last hope.

**Comment by jhonnyoddball** [comment]
@RookTKO We have deprecated a lot of react components. You should be using the new web components. In this case, you should be using `va-textarea`. [There's documentation in the storybook about it.](https://design.va.gov/storybook/?path=/docs/components-va-textarea--default)

If you still have issues, please let me know.

**Comment by RookTKO** [comment]
@jhonnyoddball just as a fyi, I looked at it and read the info yall put out and never was it a 100% must to migrate to web components from what I wrote, unless I have bad reading comprehension, and textarea is not one of the deprecated react components.
<img width="223" alt="Screen Shot 2023-01-25 at 8 06 21 AM" src="https://user-images.githubusercontent.com/16005493/214614264-dbbf71ac-8a01-48f3-89fb-fa5897b8ec8d.png">

So to me this seems like a false flag.

Are there any steps in particular that need to be done to get the web components working with React or should installation be as easing as importing the new components with the new packages?
